routes/freelance/router.js

```diff
@@ -240,6 +240,114 @@ router.put('/:freelanceid/availability', function(req, res, next) {
   }
 });
 
+// PUT freelance/:freelanceid/edit/
+router.put('/:freelanceid/edit', function(req, res, next) {
+  // TODO must Add freelance in the tags reference.
```

Comment:
looks like its done at line 306 - but if not its fine since we dont use tags like we planned to at the beginning so